code/game/objects/items/holy_weapons.dm

```diff
@@ -58,6 +58,33 @@
 	hitsound = 'sound/weapons/sear.ogg'
 	damtype = BURN
 	attack_verb = list("punched", "cross countered", "pummeled")
+	
+/obj/item/nullrod/healhand
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	name = "hand of healing"
+	desc = "This hand of yours glows with a medical power!"
+	force = 1
+	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
+	w_class = WEIGHT_CLASS_HUGE
+	hitsound = 'sound/weapons/sear.ogg' //remind me to add healing sounds :)
+	damtype = BURN
+	attack_verb = list("healed", "mended", "revitalized")
+
+/obj/item/nullrod/healhand/afterattack(atom/A, mob/user, proximity)
+	if(!proximity)
+		return
+	getBruteLoss(A)
+	user.adjustBruteLoss(brute/2)
+	A.adjustBruteLoss(-brute)
+	var/mob/living/carbon/human/H = A
+	if(!istype(H))
```

Comment:
replace 80+81 with `if(ishuman(A))` new line `var/mob/living/carbon/human/healmepls2 = A` then indent the things below and remove the return.

Reply:
oh and replace `H` with `healmeplsiamhuman`

Reply:
clown car names but i'll do it!

Reply:
I mean you can replace the var names with whatever but I prefer to make them blatantly obvious with a side of hilarity!